modules/collect/src/main/java/com/opengamma/strata/collect/result/Result.java

```diff
@@ -583,6 +585,19 @@ public boolean isSuccess() {
     return value != null;
   }
 
+  /**
+   * Executes the given consumer if the result represents a successful call and has a result available.
+   *
+   * This is the opposite of {@link #ifFailure(BiConsumer)}.
```

Comment:
No need for this - it is self explanatory. And if it were here, it would need `<p>` above it.